backends/p4tools/modules/testgen/targets/bmv2/backend/protobuf/protobuf.cpp

```diff
@@ -293,13 +293,13 @@ traces: "{{trace_item}}"
 
 input_packet {
   packet: "{{send.pkt}}"
-  port: "{{send.ig_port}}"
+  port: {{send.ig_port}}
```

Comment:
I don't know protobuf, but this change seems different.

Reply:
The protobuf spec now expects an integer instead of a string, this is why this change was necessary.